gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -313,7 +343,7 @@ Use Helm to deploy Flex Gateway to a node in your cluster.
 // end::k8s-install-flex-helm-chart-intro-local[]
 // PLS don't move k8s-mode-overview
 // tag::k8s-mode-overview[]
-For information about deployments when using {reg-mode} Mode, see xref:flex-gateway-k8-overview.adoc[].
```

Comment:
sentence below points to [this k8-related doc on deployments in connected vs. local mode](https://dev-docs-internal.kqa.msap.io/beta-flex-gateway-get-well/gateway/1.4/flex-gateway-k8-overview). Is that info true for openshift clusters? I'm wondering if the scope of that page is broader than k8. If so, perhaps it could change to say it applies to k8 and openshift clusters.